logic/service/raw-data-service.js

```diff
@@ -90,7 +113,48 @@ exports.RawDataService = DataService.specialize(/** @lends RawDataService.protot
      */
     fetchObjectProperty: {
         value: function (object, propertyName) {
-            return this.nullPromise;
+            var blueprint = this._bluePrintForObject(object),
+                propertyBlueprint = blueprint && blueprint.propertyBlueprintForName(propertyName),
+                propertyDescriptor = propertyBlueprint && propertyBlueprint.valueDescriptor;
+            return propertyDescriptor ? this._fetchObjectPropertyWithDescriptor(object, propertyName, propertyDescriptor) :
+                                        this.nullPromise;
+        }
+    },
+
+    _fetchObjectPropertyWithDescriptor: {
+        value: function (object, propertyName, propertyDescriptor) {
+            var service = this.rootService;
+            return this._blueprintTypeForValueDescriptor(propertyDescriptor).then(function (type) {
+                var selector = DataSelector.withTypeAndCriteria(type, {
+                    source: object,
+                    relationshipKey: propertyName
+                });
+                return service.fetchData(selector);
+            });
+        }
+    },
+
+    _bluePrintForObject: {
```

Comment:
we shouldn't use the word blueprint, but use descriptor, so _descriptorForObject here